dashboard/test/factories/factories.rb

```diff
@@ -422,6 +422,34 @@
     login_type 'email'
 
     initialize_with {Section.new(attributes)}
+
+    factory :mega_section do
+      after(:create) do |mega_section|
+        200.times do
+          student = create :student
+          mega_section.students << student
+        end
+        script = create :script
+        mega_section.script = script
+        i = 0
+        100.times do
```

Comment:
nit - you can save yourself a couple of lines of code by making this loop a for loop, ala
```
for i in 1..100 do
   <stuff that uses i>
end
````
This will get rid of the need to declare i, or increment it. I also think this is generally speaking the more idiomatic way to write a loop that iterates a a set number of time and accesses the iteration count in the loop.

Reply:
You can also do:
```
100.times do |i|
...
```
Too many ways to do stuff in ruby 😅